src/control/Edit/Edit.js

```diff
@@ -65,35 +65,35 @@ class Edit extends Control {
         tooltip: 'Draw a Polygon',
         draw: 'Polygon',
         element: drawButtonsDiv,
       },
       {
         name: 'line',
-        label: '<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" height="24" viewBox="0 0 24 24" width="24"><path d="M23,8c0,1.1-0.9,2-2,2c-0.18,0-0.35-0.02-0.51-0.07l-3.56,3.55C16.98,13.64,17,13.82,17,14c0,1.1-0.9,2-2,2s-2-0.9-2-2 c0-0.18,0.02-0.36,0.07-0.52l-2.55-2.55C10.36,10.98,10.18,11,10,11s-0.36-0.02-0.52-0.07l-4.55,4.56C4.98,15.65,5,15.82,5,16 c0,1.1-0.9,2-2,2s-2-0.9-2-2s0.9-2,2-2c0.18,0,0.35,0.02,0.51,0.07l4.56-4.55C8.02,9.36,8,9.18,8,9c0-1.1,0.9-2,2-2s2,0.9,2,2 c0,0.18-0.02,0.36-0.07,0.52l2.55,2.55C14.64,12.02,14.82,12,15,12s0.36,0.02,0.52,0.07l3.55-3.56C19.02,8.35,19,8.18,19,8 c0-1.1,0.9-2,2-2S23,6.9,23,8z"/></svg>',
+        label: '<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="m20 6.0c-1.1 0-1.9 0.9-1.9 2 0 0.2 0 0.4 0.1 0.6l-3.45 3.66c-0.19-0.06-0.4-0.1-0.6-0.1s-0.39 0.03-0.6 0.1l-2.43-2.583c0.059-0.198 0.089-0.404 0.089-0.612 0-1.1266-0.861-2.04-1.923-2.04-0.51 0-0.999 0.2149-1.359 0.5975-0.361 0.3826-0.564 0.9015-0.564 1.4425 0 0.208 0.031 0.414 0.089 0.612l-4.036 4.282c-0.187-0.062-0.381-0.094-0.577-0.094-0.51 0-0.999 0.215-1.36 0.597-0.36 0.383-0.563 0.902-0.563 1.443 0 1.127 0.861 2.04 1.923 2.04s1.92-0.913 1.923-2.04c0-0.208-0.03-0.414-0.09-0.612l4.036-4.282c0.187 0.062 0.381 0.094 0.577 0.094s0.391-0.032 0.58-0.09l2.433 2.58c-0.06 0.2-0.089 0.4-0.09 0.61 0 0.541 0.203 1.06 0.56 1.44 0.36 0.383 0.85 0.6 1.36 0.6 1.062 0 1.923-0.913 1.923-2.04 0-0.21-0.03-0.414-0.09-0.6l3.45-3.66c0.187 0.062 0.381 0.093 0.576 0.093 1.06 0 1.92-0.913 1.92-2.04 0-1.127-0.9-2.04-1.92-2.04z" fill="currentColor"/></svg>',
         tooltip: 'Draw a Line',
         draw: 'LineString',
         element: drawButtonsDiv,
       },
       {
         name: 'point',
-        label: '\u2022',
+        label: '<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><circle cx="12" cy="12" r="3" stroke-width="2"/></svg>',
         tooltip: 'Draw a Point',
         draw: 'Point',
         element: drawButtonsDiv,
       },
       {
         name: 'circle',
-        label: '<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"/></svg>',
+        label: '<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><circle cx="12" cy="12" r="10" stroke-width="2.5" stroke-opacity="1" fill="#dcdcdc" stroke="currentColor" /></svg>',
```

Comment:
Same as "line" and "point" above... do we need to change this?

Reply:
Maybe not... this one at least I did make a little smaller filesize-wise, but probably not enough to matter. This probably comes back to the same question as above about how important the fill is to the overall goal of these UX changes.

Reply:
In my updated commits I kept the changes to the `circle` icon since they make the overall size smaller and they add the grey background discussed above.